dnf/dnf-main.c

```diff
@@ -339,6 +342,16 @@ main (int   argc,
       if (disable_plugins_loading)
         dnf_context_set_plugins_all_disabled (disable_plugins_loading);
 
+      if (enable_disable_plugin_used && dnf_context_get_plugins_all_disabled ())
+        {
+          if (disable_plugins_loading)
+            g_print ("Loading of plugins is disabled by command line argument \"--noplugins\". "
+                     "Use of \"--enableplugin\" and \"--disableplugin\" has no meaning.\n");
+          else
+            g_print ("Loading of plugins is disabled by configuration file. "
+                     "Use of \"--enableplugin\" and \"--disableplugin\" has no meaning.\n");
```

Comment:
@Conan-Kudo hey, these messages are a bit rough... while they are perfectly understandable, we are trying to make a professional-looking product and these could use some improvements. Not all of us are proficient enough in English to do this. I try to do my best on this, but my English is certainly lacking as well.

I'm sure you're not here to do corrector work, but since you're merging the PRs, could you check the messages in them and help make them better? You're actually the only one among us perfectly capable of doing that. :slightly_smiling_face: Cheers!

Reply:
Believe it or not, I missed that! Let me look over them and see if I can make them better...

Reply:
We probably should just change `has no meaning` to `has no effect`. Otherwise it's pretty much fine.

Reply:
Okay :slightly_smiling_face: yeah that one stood out to me. So one of you guys please fix the message, @jrohel wanna do it?

Thanks Neal.